doc/data/messages/u/useless-super-delegation/details.rst

```diff
@@ -1 +1 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
+It is enough to not implement the given method and use the parent class's implementation.
```

Comment:
https://pylint.pycqa.org/en/latest/user_guide/messages/warning/useless-super-delegation.html

This is already in the normal description I think.